src/views/view_game.py

```diff
@@ -43,6 +48,21 @@ def init_window(self):
         dirty = all_sprites.draw(self.window)
         pygame.display.update(dirty)
 
+    def init_game_btns(self):
+        # Display game buttons area
+        cfg_btns = self.view_config['window']['game_buttons']
+        pygame.draw.rect(self.window, cfg_btns['color'],
+                         (cfg_btns['x'], cfg_btns['y'],
+                          cfg_btns['width'], cfg_btns['height']))
+
+        self.quit_btn = Button(pos=(1100, 610),
+                          width=80,
+                          height=80,
+                          text='Quit',
+                          background=(180, 180, 180),
+                          command=lambda x: print('a'))
+        self.quit_btn.display(self.window)
```

Comment:
This is not related to this PR but I really don't like this.
self.window.blit(self.quit_btn, (x, y)) seems way more logical

Reply:
`Button` class was implemented to have the possibility to do like that, and you don't have to give position, it uses directly `Button` attributes. For me `display` method is good !